container-images/mds-agency/server.ts

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
If we delete all of the `index.ts` files then `server.ts` could become `index.ts`